spring-oracle-example/src/main/resources/application.yml

```
@@ -0,0 +1,17 @@
+    url: ${SPRING_DATASOURCE_URL}
+    username: ${SPRING_DATASOURCE_USERNAME}
+    password: ${SPRING_DATASOURCE_PASSWORD}
+    driver-class-name: ${SPRING_DATASOURCE_DRIVER}
+
+  jpa:
+    hibernate:
+      ddl-auto: ${SPRING_JPA_HIBERNATE_DDL_AUTO}
+    show-sql: ${SPRING_JPA_SHOW_SQL}
+    properties:
+      hibernate:
+        dialect: ${SPRING_HIBERNATE_DIALECT}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)